routemaster/config/model.py

```diff
@@ -78,18 +78,19 @@ class ContextNextStates(NamedTuple):
     """Defined a choice based on a path in the given `label_context`."""
     path: str
     destinations: Iterable[ContextNextStatesOption]
+    default: str  # noqa: E704
```

Comment:
For my understanding, since I can't find this despite searching, what error is `E704`? (I'd really like to see `flake8` support pylint-style error names, but alas)

Reply:
I've added a comment, but it's basically a bug somewhere in flake8 or pycodestyle. I tried to create a failing test case for it in pycodestyle, but wasn't able to reproduce the failure (I'm not sure I understand their test case DSL format) and also that DSL doesn't seem to support differences between Python 2 and 3, and this is caused by the type annotation I believe.

My best guess is that this matches a regex that looks roughly like `^\s+def.+:.$`